webpack/scenes/ContentViews/Details/Versions/VersionDetails/ContentViewVersionDetailsTable.js

```diff
@@ -24,33 +34,35 @@ const ContentViewVersionDetailsTable = ({ tableConfig, repositories }) => {
     fetchItems,
     columnHeaders,
     disableSearch,
-  } = tableConfig;
+  }, repositories,
+}) => {
+  const ALL_REPOSITORIES = __('All Repositories');
+  const [searchQuery, updateSearchQuery] = useState('');
+  const [open, setOpen] = useState(false);
+  const { library_instance_id: urlParamId } = useUrlParams();
```

Comment:
I think I misunderstood your question earlier around this around showing repository_id vs library_instance_id..I think the param name here as repository Id makes more. The value should be library_instance_id.

Reply:
Okay, so the the parameter shown in the path to the user should say '&repository_id=123'?

Reply:
Fixed